docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "vestiging",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "contactpersoonRol": {
+            "naam": "acting subject name"
+        },
+        "indicatieMachtiging": "gemachtigde",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "12345678",
+            "vestigingsNummer": "123456789012"
+        },
+        "authenticatieContext": {
+        "source": "eherkenning",
+        "levelOfAssurance": "urn:etoegang:core:assurance-class:loa2plus",
+        "representee": {
+            "identifierType": "kvkNummer",
+            "identifier": "99998888"
+        },
+        "actingSubject": "4B75A0EA107B3D36C82FD675B5B78CC2F181B22E33D85F2D4A5DA63452EE3018@2D8FF1EF10279BC2643F376D89835151"
+        "mandate": {
+            "services": [{
+                "id": "urn:etoegang:DV:00000001002308836000:services:9113",
+                "uuid": "34085d78-21aa-4481-a219-b28d7f3282fc"
+            }]
+            }
+        }
+    }
+
+
+**eHerkenning - belanghebbende**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "niet_natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Bedrijf dat de intermediair machtigt",
+        "indicatieMachtiging": "machtiginggever",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "99998888"
+        },
+        "authenticatieContext": null
+    }
+
+
+Query patterns
+--------------
+
+Below are examples how to request zaken, authorized by different parties.
+
+**DigiD**
+
+1. Show me my cases (based on my BSN) opened for myself:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=eigen
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "initiator"
+    of the case.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=eigen
+             &rol__omschrijvingGeneriek=initiator
+
+
+2. Show me my cases (based on my BSN) opened by an authorizee on my behalf:
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__machtiging=machtiginggever
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "belanghebbende"
+    of the case.
+
+    .. code::
+
+        GET /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<own_bsn>
+             &rol__omschrijvingGeneriek=belanghebbende
```

Comment:
shouldn't that be `representee_bsn`? I find this a bit confusing compared to the above query - what's the semantic difference?